src/index.js

```diff
@@ -224,7 +106,13 @@ function getValueDoppelganger(key, originalValue) {
   const valueConverter = propertyValueConverters[key]
   let newValue
   if (valueConverter) {
-    newValue = valueConverter(importantlessValue)
+    newValue = valueConverter({
+      value: importantlessValue,
+      valuesToConvert,
+      isRtl: true,
```

Comment:
could you remind me, when is `isRtl` false?

Reply:
In the current implementation - never. However, the idea behind this refactoring is to allow other libs to use the core of rtl-css-js for other approaches to authoring bidirectional styels, specifically the one outlined in the [CSSWG's Logical Properties and Values Level 1 proposal](https://www.w3.org/TR/css-logical-1/).
In that scenario, an author will write `paddingStart`, which, depending on whether `isRtl` is `true` or `false`, will be converted to `paddingRight` or `paddeingLeft`, respectively.